stdlib/public/Platform/Windows/SwiftWinDbg/SwiftWinDbg.cpp

```diff
@@ -0,0 +1,109 @@
+#define WIN32_LEAN_AND_MEAN // Exclude rarely-used stuff from Windows headers
```

Comment:
Is there any way to test this code?

Reply:
I can't think of an obvious way to test this, besides manual testing in WinDbg. What I could do is create a new function that extracts a string when given the whole string structure. Then I could make unit tests for that function. Do you want me to do that?

Reply:
Okay.. just asking.